### ISSUE TITLE
Question about temperature / logit_scale parameters with two CLIP losses

### ISSUE THREAD

**Issue by zch0414** [google-deepmind/tips#6]
Hi, thanks a lot for the great work and the clean implementation.

I have a question regarding how the temperature (logit_scale) is handled when two CLIP losses are computed within a single training step. Specifically, do you use a single shared temperature that is updated by both CLIP losses, or do you maintain separate temperature parameters for each loss?

Thanks

**Comment by koertchen** [comment]
Thanks for the question and sorry about the delay: there are 2 separate temperature parameters, one for each loss.
